kolibri/plugins/learn/assets/src/vue/content-card/index.vue

```diff
@@ -78,8 +78,10 @@
         };
       },
       thumb() {
-        const url = this.thumbnail ? this.thumbnail : require(`./images/default_thumbnail.png`);
-        return `url(${url})`;
+        if (this.thumbnail) {
+          return `url(${this.thumbnail})`;
+        }
+        return ``;
```

Comment:
I would prefer this line was removed, and that `thumb` was therefore `undefined`.